documentation/openapi.yaml

```diff
@@ -0,0 +1,119 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0-oas3
+  title: Kubportal Backend API
+paths:
+  /statistics/{stat}:
+    get:
+      parameters:
+        - in: path
+          name: stat
+          required: true
+          schema:
+            type: string
+          description: The statistic type being fetched
+      responses:
+        '200':
+          description: A single statistic value
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  value:
+                    type: string
+                    example: 1.0.0
+  /webapps/:
+    get:
+      responses:
+        '200':
+          description: A list of web applications allowed for the user
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    link_name:
+                      type: string
+                      example: kuchen
+                    link_url:
+                      type: string
+                      example: 'https://mango-saft.com'
+                    link_show:
+                      type: boolean
+                      example: false
+
+  /users/{uid}:
+    get:
+      parameters:
+        - in: path
+          name: uid
+          required: true
+          schema:
+            type: string
```

Comment:
That can be an INT.

Reply:
okay, I remembered some auto-generated django - ids and wasn't sure. thanks.